CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Welcome!
```

Comment:
np. i think I stole this from Crayfish or Chullo somewhere, so I'll pull from another source and update those as well.